server/resources/translations/common.en.json

```diff
@@ -9,6 +10,10 @@
   "incidentCategorized": "The incident was categorized as {{ incidentCategory }}.\n\nThank you. This session is now complete. (You don't need to respond to this message.)",
   "incidentCategoryInvalid": "Sorry, the incident type wasn't recognized. Please try again.",
   "incidentCategoryRequest": "Once you have responded, please reply with the number that best describes the incident:\n{{ incidentCategories }}",
+  "resetNoticeToOtherResponders": "Another Responder has acknowledged this request and decided to reset the Brave Sensor. (You don't need to respond to this message.)",
```

Comment:
I'd take out the word "acknowledged" in this string. To me, acknowledged sounds too much like the term that we use when they respond "ok" to an alert. Also, I like how you have explained the consequences of the rest to the Responder. I think that that would also be useful for the OtherResponders. So how about something like:

 "Another Responder has reset the Brave Sensor.\n\nPlease note that after a reset, the sensor will not accurately monitor for stillness until about ten minutes have passed. If you have not checked on this location yet, please do so.\n\nThis session is now complete. (You don't need to respond to this message.)"

Reply:
Done! All text is pending opinions from the team; will discuss with them.